CODEOWNERS

```diff
@@ -453,8 +453,9 @@
 /include/toolchain/                       @andrewboie @andyross
 /include/zephyr.h                         @andrewboie @andyross
 /kernel/                                  @andrewboie @andyross
-/lib/fnmatch/                             @carlescufi
+/lib/fnmatch/                             @carlescufi @jakub-uC
 /lib/gui/                                 @vanwinkeljan
+/lib/getopt/                              @jakub-uC
```

Comment:
need to move this to some common place, having a lib per file (see fnmatch) is an overkill. Lets move all of those one lib level. This was discussed some time ago in the API meeting.

Reply:
fixed